README.md

```diff
@@ -166,6 +166,7 @@ Thanks goes to these Wonderful People. Contributions of any kind are welcome! :g
     <td align="center"><a href="https://github.com/supzi-del"><img src="https://avatars.githubusercontent.com/u/78655439?v=4" height="120px" width="120px"/><br/><sub><b>Supritha</b></sub></a></td>
     <td align="center"><a href="https://github.com/Vedant-Jayesh-Oza"><img src="https://avatars.githubusercontent.com/u/75005433?v=4" height="120px" width="120px"/><br/><sub><b>Vedant Jayesh Oza</b></sub></a></td>
     <td align="center"><a href="https://github.com/jigar-sable"><img src="https://avatars.githubusercontent.com/u/64949957?v=4" height="120px" width="120px"/><br/><sub><b>Jigar Sable</b></sub></a></td>
```

Comment:
Hey @Sidhved this be fine. I'll however suggest you to shift yours and @jigar-sable 's columns to the next row in the table for better visibility on both mobile and desktop.

Reply:
Ohkk sure I'll get that resolved